lib/xnvmec.c

```diff
@@ -939,6 +939,33 @@ static struct xnvmec_opt_attr xnvmec_opts[] = {
 		.name = "\nWith <args> for backend:\n",
 		.descr = "",
 	},
+
+	{
+		.opt = XNVMEC_OPT_AD,
+		.vtype = XNVMEC_OPT_VTYPE_NUM,
+		.name = "ad",
+		.descr = "If set, deallocate ranges when calling Dataset "
```

Comment:
I don't think there's any need for specifying that this (and the 2 other opts below) is for DSM.
This will be clear from the context when they're listed in `xnvme dsm --help`

Reply:
So you want to remove the "... when calling Dataset Management command (DSM)" part? I agree

Reply:
Exactly

Reply:
Done